specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/preview/2019-05-01/NasCluster.json

```diff
@@ -0,0 +1,207 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Nas Cluster operation endpoints and objects.",
+    "title": "FabricAdminClient",
+    "version": "2019-05-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Fabric.Admin/fabricLocations/{location}/nasClusters/{nasCluster}": {
+      "get": {
```

Comment:
>get [](start = 7, length = 3)

How are these NAS clusters created?

Reply:
It is created by a PowerShell command to invoke the Azure Stack Hub Enterprise Cloud Engine to execute the AddCluster action plan. There are a lot of operations need to be done before the resource are available so user would not be able to create this resource by ARM API.

Reply:
Thanks for explaining.